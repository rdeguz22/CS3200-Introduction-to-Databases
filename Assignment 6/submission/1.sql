SELECT mg.GENRE, COUNT(mg.GENRE) AS MOVIE_COUNT
FROM MOVIES_GENRES AS mg
GROUP BY mg.GENRE
HAVING COUNT(mg.GENRE) = (
    SELECT MAX(MOVIE_COUNT)
    FROM (
        SELECT COUNT(GENRE) AS MOVIE_COUNT
        FROM MOVIES_GENRES
        GROUP BY GENRE
    ) AS subquery
)
ORDER BY GENRE
