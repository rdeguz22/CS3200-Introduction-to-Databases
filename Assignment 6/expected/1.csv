GENRE,MOVIE_COUNT
Drama,17
Thriller,17
